dev/io.openliberty.microprofile.telemetry.1.0.internal/bnd.bnd

```diff
@@ -31,15 +31,14 @@ src: src, resources
 
 -dsannotations: \
   io.openliberty.microprofile.telemetry.internal.cdi.SPIMetaData,\
-  io.openliberty.microprofile.telemetry.internal.cdi.OpenTelemetryProducer,\
-  io.openliberty.microprofile.telemetry.internal.rest.TelemetryClientFilter,\
-  io.openliberty.microprofile.telemetry.internal.rest.TelemetryContainerFilter,\
-  io.openliberty.microprofile.telemetry.internal.rest.TelemetryClientAsyncTaskWrapper
+  io.openliberty.microprofile.telemetry.internal.rest.TelemetryClientAsyncTaskWrapper,\
```

Comment:
Copyright date

Reply:
Drat, I thought I checked for all of these.

Reply:
Fixed